packages/url-loader/src/plugins/cropping.ts

```diff
@@ -1,18 +1,23 @@
 import * as parameters from '../constants/parameters';
 
-import { PluginSettings, PluginOverrides } from '../types/plugins';
+import { PluginSettings, PluginResults } from '../types/plugins';
 
-const cropsAspectRatio = [ 'crop', 'fill', 'lfill', 'fill_pad', 'thumb' ];
-const cropsGravityAuto = [ 'crop', 'fill', 'lfill', 'fill_pad', 'thumb' ];
+const cropsAspectRatio = [ 'auto', 'crop', 'fill', 'lfill', 'fill_pad', 'thumb' ];
+const cropsGravityAuto = [ 'auto', 'crop', 'fill', 'lfill', 'fill_pad', 'thumb' ];
 const cropsWithZoom = ['crop', 'thumb'];
 
 const DEFAULT_CROP = 'limit';
 
 export const props = {
   aspectRatio: parameters.aspectRatio.schema.optional(),
+  baseAspectRatio: parameters.baseAspectRatio.schema.optional(),
```

Comment:
question: is it intentional to have both base and aspectRatio here?

Reply:
yes as they provide 2 different sets of functionality like mentioned here: https://github.com/colbyfayock/cloudinary-util/pull/139#discussion_r1489568044

let me know if there's something i can clarify in the PR description as i was hoping to catch the reasoning there

Reply:
Good one, I think I just missed it. Thanks for clarification! :)